project/Dependencies.scala

```diff
@@ -201,6 +201,15 @@ object Dependencies {
         Test.logback,
         Test.scalatestJUnit)
```

Comment:
any of the test dependencies that can be removed here now?

Reply:
It still has a unit test, but I got rid of a few.